PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((float)numberCorrectAnswers / (float)numberOfQuestions * 100) + "%";
```

Comment:
Only thing I might have done would have been something like the following so that if it isn't a whole number when done, you don't end up with any weird decimal answers.

```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```